test/k8sT/microscope.go

```diff
@@ -56,14 +55,19 @@ var _ = Describe(microscopeTestName, func() {
 		microscopeErr, microscopeCancel := kubectl.MicroscopeStart()
 		Expect(microscopeErr).To(BeNil(), "Microscope cannot be started")
 
-		time.Sleep(time.Second * 10)
+		Eventually(func() bool {
```

Comment:
Hi, 

Please don't use Eventually, it does not play nice with BeforeAll and AfterAll, please use `helpers.WithTimeout()` instead.

Reply:
Done